client/src/main/java/com/networknt/client/ssl/X509TrustManagerDecorator.java

```diff
@@ -0,0 +1,29 @@
+package com.networknt.client.ssl;
+
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.X509ExtendedTrustManager;
+import javax.net.ssl.X509TrustManager;
+
+public class X509TrustManagerDecorator {
+	public static TrustManager[] decorate(TrustManager[] trustManagers, boolean checkIdentity, String trustedNames) {
+		if (null!=trustManagers && trustManagers.length>0) {
+			TrustManager[] decoratedTrustManagers = new TrustManager[trustManagers.length];
```

Comment:
What's this used for?

Reply:
This is used to convert X509TrustManagers loaded by service loaders to our customized ones.